.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "3rdparty/googletest"]
 	path = 3rdparty/googletest
 	url = https://github.com/google/googletest
+[submodule "3rdparty/opencv/opencv"]
+	path = 3rdparty/opencv/opencv
```

Comment:
Why it is opencv/opencv?

Reply:
github does not allow to make 2 submodules in one folder

Reply:
Actually, it is allowed. Please, recheck